src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -0,0 +1,80 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { Inscription, InscriptionResponseItem } from '@shared/models/inscription.model';
+
+import { createNullArrayOfLength } from '@app/common/utils';
+import { createCounter } from '@app/common/utils/counter';
+import { QueryPrefixes } from '@app/query/query-prefixes';
+import { useCurrentAccountIndex } from '@app/store/accounts/account';
+import { useCurrentTaprootAccountKeychain } from '@app/store/accounts/blockchain/bitcoin/taproot-account.hooks';
+import { useCurrentNetwork } from '@app/store/networks/networks.selectors';
+
+import { getTaprootAddress } from './utils';
+
+const stopSearchAfterNumberAddressesWithoutOrdinals = 20;
+const addressesSimultaneousFetchLimit = 5;
+
+// max limit value in Hiro API
+const inscriptionsLimit = 60;
+
+async function fetchInscriptions(addresses: string) {
```

Comment:
Might be cleaner to accept an array here, and put the logic for turning into query param in this function?

Reply:
reason why I haven't put it here is that `fetch<something>` function should be responsible for only fetching, without some other logic in it